spec/rails_helper.rb

```diff
@@ -0,0 +1,24 @@
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path('../config/environment', __dir__)
+
+abort('The Rails environment is running in production mode!') if Rails.env.production?
+
+require 'rspec/rails'
+require 'spec_helper'
+
+Dir[Rails.root.join('spec/support/*.rb')].each { |file| require file }
+
+SimpleCov.start
+
+begin
+  ActiveRecord::Migration.maintain_test_schema!
+rescue ActiveRecord::PendingMigrationError => e
+  puts e.to_s.strip
+  exit 1
+end
+
+RSpec.configure do |config|
+  config.use_transactional_fixtures = true
+  config.infer_spec_type_from_file_location!
+  config.filter_rails_from_backtrace!
+end
```

Comment:
I would add 

ActiveRecord::Migration.maintain_test_schema!
WebMock.disable_net_connect!(allow_localhost: true) # (for this we need to install webmock gem)

Reply:
The `ActiveRecord::Migration.maintain_test_schema!` configuration it's already there. It's set by default when installing rspec

Reply:
nice to know that, thanks!